docarray/array/storage/redis/backend.py

```diff
@@ -94,25 +101,19 @@ def _init_storage(
         elif isinstance(_docs, Document):
             self.append(_docs)
 
-    def _build_client(self):
-        client = Redis(
-            host=self._config.host,
-            port=self._config.port,
-            **self._config.redis_config,
-        )
-
-        if self._config.update_schema:
-            if self._config.index_name.encode() in client.execute_command('FT._LIST'):
-                client.ft(index_name=self._config.index_name).dropindex()
+    def _build_index(self, rebuild: bool = False):
```

Comment:
I am not familiar with the inner workings of redis, could you very briefly explain why this `rebuild` argument is needed?

Reply:
RedisConfig has boolean `update_schema` to decide whether to update Redis Search index schema during initialization. I want the function `_build_index` to be used by both `_init_storage` and rebuild index of `_clear_storage`. This `rebuild` argument is for the latter situation.